File: TypeScript+React/Fotballportalen(exam2021)/fotballportalen/src/interfaces/ITeam.ts
export interface ITeam {
    //Retter seg mot ETT spesifikt objekt
    id?: string;
    name?: string;
    image?: string;
    stadium?: string;
    capacity?: number;
    manager?: string;
    founded?: number;
    alias?: string;
    team_id?: number; //1-20
}

File: TypeScript+React/Fotballportalen(exam2021)/FpApi/Interfaces/ITeam.cs
namespace FpApi.Interfaces
{
    public interface ITeam
    {
        string id { get; set; }
        string alias {get;set;}
        int capacity { get; set; }
        int founded { get; set; }
        string image { get; set; }
        string manager { get; set; }
        string name { get; set; }
        string stadium { get; set; }
        int team_id { get; set; }
    }
}